### PR TITLE
Add PageTabView normal / selected style

### PR DESCRIPTION
**Pull request by eJamesLin**
### About
- Add PageTabView normal / selected color and font style

### CODE REVIEW

File: LineSDK/LineSDK/SharingUI/PageTabView.swift [hunk 1]
```diff
@@ -187,6 +203,11 @@ class PageTabView: UIView {
     // This only update the `selectedIndex` property and update style when neccessary.
     func updateSelectedIndex(_ index: Int) {
         selectedIndex = index
+
+        // update tabs style
+        tabs.enumerated().forEach { (i, tabView) in
+            tabView.isSelected = (i == selectedIndex)
+        }
```
Comment (onevcat) [inline LineSDK/LineSDK/SharingUI/PageTabView.swift]: Maybe we can move this to `selectedIndex` setter and remove the `updateSelectedIndex(_ index: Int)`.

Comment (eJamesLin) [reply LineSDK/LineSDK/SharingUI/PageTabView.swift]: Good idea. Let me update it.

File: LineSDK/LineSDK/SharingUI/PageTabView.swift [hunk 2]
```diff
@@ -58,6 +67,13 @@ class PageTabView: UIView {
                 textLabel.widthAnchor.constraint(lessThanOrEqualTo: widthAnchor, multiplier: 0.8)
             ])
         }
+
+        override var isSelected: Bool {
+            didSet {
+                textLabel.font = isSelected ?Design.selectedTitleFont :Design.titleFont
+                textLabel.textColor = isSelected ?Design.selectedTitleColor :Design.titleColor
```
Comment (onevcat) [inline LineSDK/LineSDK/SharingUI/PageTabView.swift]: Please change this `isSelected ?Design.selectedTitleColor :Design.titleColor`

to `isSelected ? Design.selectedTitleColor : Design.titleColor`. It is very rare to use [the ternary operator](https://en.wikipedia.org/wiki/%3F:) like this and I've never seen it. Is there any article or convention to use it like this?

File: LineSDK/LineSDK/SharingUI/PageTabView.swift (hunk 1 above)
Comment (onevcat) [reply LineSDK/LineSDK/SharingUI/PageTabView.swift]: I am f*cked up. Please ignore thi.

File: LineSDK/LineSDK/SharingUI/PageTabView.swift [hunk 3]
```diff
@@ -217,7 +238,7 @@ class PageTabView: UIView {
     func resetUnderline() {
         currentProgress = 0
         currentDiff = 0
-        selectedIndex = 0
+        updateSelectedIndex(0)
```
Comment (onevcat) [inline LineSDK/LineSDK/SharingUI/PageTabView.swift]: Let's move it to `init`. Updating selected index in an underline related method is a bit strange and may cause issues in the future.

**Pull request merged**